cms/authenticate.lib.php

```diff
@@ -137,9 +139,9 @@ function getGroupIds($userId) {
 	else
 		$groups[] = 1;
 	$groupQuery = 'SELECT `group_id` FROM `' . MYSQL_DATABASE_PREFIX . 'usergroup` WHERE `user_id` = \'' . escape($userId)."'";
-	$groupQueryResult = mysql_query($groupQuery) or die(mysql_error());
-	while ($groupQueryResultRow = mysql_fetch_row($groupQueryResult))
-		$groups[] = $groupQueryResultRow[0];
+	$groupQueryResult = $pdb->query($groupQuery);
```

Comment:
@nean In _init function, I'm using prepare. So there is actually no need to escape/quote http://php.net/manual/en/pdo.prepare.php